### ISSUE TITLE
What does this do that Webpack doesn't do natively?

### ISSUE THREAD

**Issue by STRML** [alexindigo/webpack-chunk-hash#8]
Webpack allows setting `hashFunction`, `hashDigest`, and `hashDigestLength` in `outputOptions` and also uses `crypto.createHash()`. 

It appears the difference is that this plugin uses the actual contents, not just all the ids included in the file, as [webpack/lib/Chunk.js](https://github.com/webpack/webpack/blob/b2a6085d9684cb8bb3c47cbcb778413844d70d53/lib/Chunk.js#L282) seems to say. Is that correct?

**Comment by alexindigo** [comment]
Yes, this plugin is trying to produce hash as close to the final bundle as it's possible in webpack universe.

**Comment by stephan-v** [comment]
> not just all the ids included in the file.

This is exactly why this plugin is a bad idea. If you are not getting a new file served to you because your hash did not change but indices within your module have, you are getting mismatches within your `manifest.js` file.
